app/src/main/assets/about-es.html

```diff
@@ -12,13 +12,13 @@
 
 <p>Creado por <i>Jeroen Carolus</i></p>
 
-<p>Para el apoyo , la informaci�n acerca de las actualizaciones y la versi�n m�s reciente : por favor vaya a la p�gina web de desarrolladores <a href='http://www.jwtc.nl/'>jwtc.nl</a>.</p>
+<p>Para el apoyo , la informaci�n acerca de las actualizaciones y la versi�n m�s reciente : por favor vaya a la p�gina web de desarrolladores <a href='http://www.jwtc.nl/'>jwtc.nl</a>.</p>
```

Comment:
I'm not sure what happened here.  I think somehow the html files were refreshed (maybe an Android update?) and they weren't  re-saved in UTF-8 format.  I put UTF-8 in the meta header, so hopefully it won't happen again.